index.js

```diff
@@ -57,9 +57,7 @@ const lookupInEntityMap = (entityMap, linkData) => {
 }
 
 const getIdsFromUrn = (urn) => {
-  // TODO: Use the regex in resource-names package when it's public.
-  const regExp =
-    /.*:spaces\/(?<spaceId>[^/]+)(?:\/environments\/(?<environmentId>[^/]+))?\/entries\/(?<entityId>[^/]+)$/
+  const regExp = /.*:spaces\/([^/]+)(?:\/environments\/([^/]+))?\/entries\/([^/]+)$/
```

Comment:
`?:` starts a non-capturing group. Do we have a test for whether we’re extracting the environment correctly?